README.md

```diff
@@ -137,17 +155,15 @@ Instantiate a new `Storj` object used to communicate with the storj network. The
 
 If you need to use authentication in your application, we strongly suggest you use the `key` method as it provides a higher level of security.
 
-Both `basicAuth` and `key` are optional, but you may only provide one or the other. If you use `basicAuth`, the library will attempt to create a public/private key pair for you and register it to your account. If you provide a `key`, this key will be used to authenticate every request moving forward.
+Both `basicAuth` and `key` are optional, but you may only provide one or the other. If you use `basicAuth`, the library will assume that you already have registered a public key with the bridge you are authenticating with. To create a public/private key pair for you and register it to your account you can use the `getKeypair` and `registerKeypair` API. If you provide a `key`, this key will be used to authenticate every request moving forward.
 
 ### `storj.on('error', function (e) ...`
 
 Emitted when the library reaches a catastrophic error that will probably prevent normal operation moving forward.
 
 ### `storj.on('ready', function () ...`
 
-Emitted when all setup is complete. If you are using `basicAuth`, validating that auth and creating a private/public key pair for your account will be an asyncronous; as soon as the `storj` object has been configured to use your `basicAuth`, `ready` will be emitted.
-
-If you are passing in a `key`, `ready` will fire immediately as there will be no asyncrnous work. Note that the `key` is not validated, if you provide an invalide `key` future requests may fail.
+Emitted when all setup is complete. As soon as the `storj` object has been configured to use your `basicAuth` or `key`, `ready` will be emitted.
```

Comment:
If we aren't doing any async for this can we just drop the `ready` event for now? `basicAuth` and `key` aren't validated until used, so we don't need to break the event loop in the constructor anymore.

Alternatively we could leave this event so the pattern of using either a callback to the constructor or waiting for the `ready` event is already being used in the community in the likely event we need to make our constructor async in the near future.